samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
+@property (nonatomic) UISlider* fontSize;
+@property (nonatomic) UITextField* x;
+@property (nonatomic) UITextField* y;
+@property (nonatomic) UITextField* w;
+@property (nonatomic) UITextField* h;
+@property (nonatomic) UITextField* numberOfLines;
+@property (nonatomic) UITextField* lineBreakMode;
+@property (nonatomic) UISwitch* autoShrink;
+@property (nonatomic) UITextField* minimumFontSize;
+@property (nonatomic) UITextField* minimumScaleFactor;
+@property (nonatomic) UILabel* label;
```

Comment:
Is this the control we're manipulating? Can we pull it out on its own above and make a comment so it's clear.